modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/LocalPartitionState.java

```diff
@@ -18,17 +18,22 @@
 package org.apache.ignite.internal.table.distributed.disaster;
 
 import org.apache.ignite.internal.partition.replicator.network.disaster.LocalPartitionStateEnum;
+import org.apache.ignite.internal.tostring.IgniteToStringInclude;
+import org.apache.ignite.internal.tostring.S;
 
 /**
  * Local partition state.
  */
 public class LocalPartitionState {
+    @IgniteToStringInclude
     public final String tableName;
 
     public final String zoneName;
 
+    @IgniteToStringInclude
```

Comment:
Is this necessary for primitives? I remember that no.

Reply:
Answered in a different thread